src/base/rss/rss_article.cpp

```diff
@@ -77,11 +93,8 @@ Article::Article(Feed *feed, const QVariantHash &varHash)
 }
 
 Article::Article(Feed *feed, const QJsonObject &jsonObj)
-    : Article(feed, jsonObj.toVariantHash())
+    : Article(feed, articleDataFromJSON(jsonObj))
 {
-    // JSON object store DateTime as string so we need to convert it
-    m_date = QDateTime::fromString(jsonObj.value(KeyDate).toString(), Qt::RFC2822Date);
```

Comment:
`m_date` will be uninitialized from this code path

Reply:
It is initialized by delegating constructor:
```c++
Article::Article(Feed *feed, const QJsonObject &jsonObj)
    : Article(feed, articleDataFromJSON(jsonObj)) // here is it!
 {
```

Reply:
oops...